app/src/styles/discover.js

```diff
@@ -80,8 +80,13 @@ const discoverStyle = StyleSheet.create({
     textAlign: 'center',
     color: '#0c604b'
   },
+  drawerMenuButton: {
+    height: '100%',
+    justifyContent: 'center'
+  },
   drawerHamburger: {
-    marginLeft: 16
+    marginLeft: 16,
+    marginRight: 16
```

Comment:
Should this be `padding` so the touch area is bigger?

Reply:
This sets the margin on the icon (which is sort of a text element) itself. The alternative would be to apply padding to the container element.

Reply:
Ah right 👍